app/controllers/registered_applications_controller.rb

```diff
@@ -0,0 +1,62 @@
+class RegisteredApplicationsController < ApplicationController
+  def index
+    @registered_applications = RegisteredApplication.all
+  end
+
+  def show
+    @registered_application = RegisteredApplication.find(params[:id])
+  end
+
+  def new
+    @user = current_user
+    @registered_application = RegisteredApplication.new
+  end
+
+  def create
+    @registered_application      = RegisteredApplication.new
+    @registered_application.name = params[:registered_application][:name]
+    @registered_application.url  = params[:registered_application][:url]
+    @user                        = current_user
+    @registered_application.user = @user
+
+    if @registered_application.save
+      redirect_to @registered_application, notice: "Registered Application was saved successfully."
+    else
+      flash.now[:alert] = "Error Registering Application. Please try again."
+      render :new
+    end
+  end
+
+  def edit
+    @user = current_user
+    @registered_application = RegisteredApplication.find(params[:id])
+  end
+
+  def update
+    @registered_application = RegisteredApplication.find(params[:id])
+
+    @registered_application.name = params[:registered_application][:name]
```

Comment:
Look into using strong params for these.

Reply:
I looked up strong params but I'm not sure I really understand what it means to use strong params. Is it something such as putting the params in one line instead of multiple lines?